Rakefile

```diff
@@ -1,5 +1,25 @@
+require 'json'
+require 'net/https'
 require 'rake/testtask'
 
+task :download do
+  LICENSES_URI = URI.parse('https://spdx.org/licenses/licenses.json')
+
+  http = Net::HTTP.new(LICENSES_URI.host, LICENSES_URI.port)
+  if LICENSES_URI.scheme == 'https'
+    http.use_ssl = true
+    http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Oh, it did for me - perhaps a different in cert bundles or similar.  Not to worry.